src/html/signin.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css" />
+    <link rel="stylesheet" href="../css/style.css">
+    <link rel="stylesheet" href="../css/style-form.css">
+    <link rel="shortcut icon" href="../../images/logo.png" />
+    <title>Register</title>
+</head>
+<style>
+    a:hover {
+        color: #d0d0d0; 
+        
+    }
+</style>
+<body id="universe">
+    <section class="form-section">
+        <h1 class="title">Sign In / Log In</h1>
+        <form id="form">
+          
+          <div class="input-control">
+            <label for="">Email</label>
+            <input type="email" placeholder="info@email.com" id="email" />
+            <i class="fas fa-check-circle"></i>
+            <i class="fas fa-exclamation-circle"></i>
+            <small>Error message</small>
+          </div>
+  
+          <div class="input-control">
+            <label for="">Password</label>
+            <input type="password" placeholder="password" id="password" />
+            <i class="fas fa-check-circle"></i>
+            <i class="fas fa-exclamation-circle"></i>
+            <small>Error message</small>
+          </div>
+          <div>
+            <h4>Create an account <a href="signup.html" style="text-decoration: none;">Click Here</a></h4>
```

Comment:
Why not just make the whole heading as a hyperlink?
`<a href="signup.html" style="text-decoration: none;"><h4>New User?</h4> </a>`